### ISSUE TITLE
Stuck at that "something went wrong :( " screen.

### ISSUE THREAD

**Issue by takscreen** [withfig/fig#2733]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 12.6.3 (21G419)

### Expected behaviour

work.

### Actual behaviour

1. installed fig
2. trying to config ie
3. tried to signup with an email instead of github.
4. after providing the MFA code from email to try to login, fig-app stuck at that "Something went wrong : (" screen on the back.

Tried:
-close and reopen app
-fig restart on terminal
-force quit and restart
-tried to go backward
-tried "fig settings" then it goes back to the "Something went wrong :(" screen.  

This "something went wrong" screen stated that no email login/sign available, that's should be fine, then perhaps, don't offer that option.  Even if the option failed, give me an option to 'go-back', i can use github to sign in.  Not easy to use for new users.

### Steps to reproduce

1. install fig
2. try to sign up with email
3. enter code from email.

### Environment

```yaml
fig-details:
  - 2.16.0
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz
  - cores: 8
  - mem: 32.00 GB
os-info:
  - macOS 12.6.3 (21G419)
environment:
  - shell: /usr/local/Cellar/bash/5.1.8/bin/bash
  - terminal: iterm
  - cwd: /Users/brandon.chan/Documents/DBT/workfolder
  - exe-path: /usr/local/bin/fig
  - install-method: brew
  - env-vars:
    - SHELL: /usr/local/bin/bash
    - __CFBundleIdentifier: com.googlecode.iterm2
    - TERM: xterm-256color
    - PATH: /Users/brandon.chan/Downloads/google-cloud-sdk/bin:/Applications/SnowSQL.app/Contents/MacOS:/Users/brandon.chan/.asdf/shims:/Users/brandon.chan/.asdf/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
```

**Comment by akadori** [comment]
> [New users will not be able to sign up for Fig's products right now while we focus on optimizing them for existing customers and addressing some needs identified to integrate Fig with AWS.](https://fig.io/blog/post/fig-joins-aws#:~:text=New%20users%20will%20not%20be%20able%20to%20sign%20up%20for%20Fig%27s%20products%20right%20now%20while%20we%20focus%20on%20optimizing%20them%20for%20existing%20customers%20and%20addressing%20some%20needs%20identified%20to%20integrate%20Fig%20with%20AWS.)

**Comment by grant0417** [comment]
Hey, we dont have signups enabled right now, for more info see here: https://fig.io/blog/post/fig-joins-aws

**Comment by takscreen** [comment]
Thanks, but the "something went wrong" setting/config saved on my mac.  Any information how I can do a 'clean' uninstall and then re-install again?   

I did a 'fig uninstall' which removed it.  When I try to reinstall with 'brew', it didn't work and prompted that I have 'already' installed(which is not true as I just uninstall it).  When I download the dmg file and install through UI, the installation seems to work.  Then when I go back to setting, the app led me back to "something went wrong".  

In short, I am not stuck at this "something went wrong" state.

If only I can 'go-back', I can sign up with github instead.  However, I am not stuck at this state with this app and not usable.

**Comment by grant0417** [comment]
You can use `brew reinstall --force --cask fig` if you want to reinstall. Also you can use `fig logout && fig quit && fig` if you want to try login in again.

**Comment by takscreen** [comment]
Thanks!!! 'fig logout' got me out the state and I can proceed with github login.  Thanks!!!
